Formula/fail2ban.rb

```diff
@@ -11,11 +11,12 @@ class Fail2ban < Formula
   end
 
   depends_on "help2man" => :build
+  depends_on "python@3.8"
```

Comment:
This will complain: alphabetical order within category, with all build deps first, then runtime

Reply:
Yes, I have realised that as well. Apparently I did a mistake by putting `python@3.8` as a build dependency only